src/azure-cli/setup.py

```diff
@@ -30,175 +30,179 @@
 
     m = re.search(r'__version__\s*=\s*[\'"](.+?)[\'"]', content)
     if not m:
-        print('Could not find __version__ in azure/cli/__main__.py')
+        print("Could not find __version__ in azure/cli/__main__.py")
```

Comment:
Why changing to double quote?

Reply:
Oof, vscode did that unexpectedly. I'll correct the PR.